### ISSUE TITLE
option to automatically apply quick fixes

### ISSUE THREAD

**Issue by DetachHead** [dart-lang/linter#3283]
**Describe the issue**
it's annoying when simple issues like this aren't automatically fixed
![image](https://user-images.githubusercontent.com/57028336/158168949-549bc365-5362-4f5d-b71a-6a62f4980584.png)

ideally i'd like to be able to configure my IDE to automatically fix simple issues like this on save

**Comment by bwilkerson** [comment]
> Doesn't Idea have a "Apply to all similar issues in file" thing?

Yes. There should be a "Convert to single quoted strings everywhere in file" option if there's more than one place that needs to be fixed.

> Not sure, does it let you do that on save

Not that I'm aware of, at least not yet for Dart. There is support for some other languages, so we'll discuss adding support for Dart.
